packages/cli/src/config/schema.ts

```diff
@@ -908,6 +908,40 @@ export const schema = {
 		},
 	},
 
+	externalStorage: {
+		s3: {
+			bucket: {
+				name: {
+					format: String,
+					default: '',
+					env: 'N8N_EXTERNAL_OBJECT_STORAGE_BUCKET_NAME',
+					doc: 'Name of the n8n bucket in S3-compatible external storage',
+				},
+				// @TODO: Is region AWS-specific?
+				region: {
+					format: String,
+					default: '',
+					env: 'N8N_EXTERNAL_OBJECT_STORAGE_BUCKET_REGION',
+					doc: 'Region of the n8n bucket in S3-compatible external storage',
+				},
+			},
+			credentials: {
```

Comment:
Just food for thought: It might be useful to have the ability to manage S3 credentials in the same way as regular n8n credentials.

Reply:
Thanks! Consolidating [here](https://linear.app/n8n/issue/PAY-769/introduce-object-store-manager-for-binary-data#comment-db61b74b).